devices/wholeBodyDynamics/WholeBodyDynamicsDevice.cpp

```diff
@@ -1363,8 +1382,17 @@ bool WholeBodyDynamicsDevice::attachAll(const PolyDriverList& p)
     ok = ok && this->attachAllVirtualAnalogSensor(p);
     ok = ok && this->attachAllFTs(p);
     ok = ok && this->attachAllIMUs(p);
-
-    ok = ok && this->setupCalibrationWithExternalWrenchOnOneFrame("base_link",100);
+    
+    if (settings.skipResetFTSensorOffset)
+    {
+        this->setFTSensorOffsetsToZero();
+        validOffsetAvailable = true;
+        ok = ok && true;
```

Comment:
It is not really clear what is the  point of this line.

Reply:
I added that redundant code to debug and forgot to remove.

Reply:
Addressed in https://github.com/robotology/whole-body-estimators/pull/72/commits/b495053958d918d01711010f7e32f1ac651fc29c